### ISSUE TITLE
Add quick starting example on the README

### ISSUE THREAD

**Issue by jordanmontt** [PolyMathOrg/DataFrame#183]
It would be nice to have some examples of how to get start in the README

**Comment by Ducasse** [comment]
Remember that in P11 you will be able to read the documentation that is located in the doc folder. 

S

**Comment by jordanmontt** [comment]
Yes, that is why we want to have a good README file with good documentation :)

**Comment by jecisc** [comment]
A quick start should be enough because there is a booklet with much more documentation provided also
